WordPress/gutenberg
# HTML API: Move into 6.2 Compat Folder since inclusion in Core

## Status

Is there any guidance on moving PHP support into `lib/compat`? I ran around like a self-driving taxi without its sensors trying to mimic what I saw, the worst kind of imitation. This is probably all wrong.

## What?

Moves the HTML Tag Processor out of `experimental` and into `combat/wordpress-6.2`

## Why?

The Tag Processor was merged into 6.2. It needs to load only when not already present from WordPress itself.

## How?

Copies files from the merged Core code into `lib/compat/wordpress-6.2` and updates the `load.php` file to conditionally load the compatibility class.

## Testing Instructions

Since code in Gutenberg relies on the Tag Processor tests should start failing if something goes wrong. Given that this is merged in Core, ensure that the files copied over are the appropriate files and do not differ from what's in Core. Ensure that they load when running on pre-6.2 versions of WordPress.
